perfact/zodbsync/commands/upload.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+from .pick import Pick
+import os
+import sys
+
+PY2 = (sys.version_info.major == 2)
+
+FOLDER_TEMPLATE = '''[
+    ('props', []),
+    ('title', ''),
+    ('type', 'Folder'),
+]
+'''
+
+JS_TEMPLATE = '''[
+    ('props',[[('id','content_type'),('type','string'),('value','text/js')]]),
+    ('title','content.min.css'),
```

Comment:
Or maybe skip writing the file system representation and letting it be read from `playback`. Simply use `mod_write` from `zodbsync.py` to directly create the Data.FS objects from `zodbsync.py`. The recorder or watcher will then create the corresponding file system structure.

Also, `text/js` is not the correct content type for Javascript. It should be `application/javascript`.

Reply:
Got it, will correct content type. Will also use `mod_format` for meta file creation, this will beautify the code.

I thought about this approach ( using `mod_write` ), but to be honest it seemed harder to implement than some simple file conversion leveraging the existing file structure. This way i could avoid diving too deep into `zodbsync.py` :wink: